localgov_base_croydon.theme

```diff
@@ -210,3 +210,22 @@ function localgov_base_croydon_preprocess_block(array &$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_field() to customize file links on
+ * localgov_publication_cover_page nodes for the field_media_document.
+ */
+function localgov_base_croydon_preprocess_field(array &$variables, string $hook): void {
+  if ($variables['element']['#field_name'] === 'field_media_document' &&
+      $variables['element']['#entity_type'] === 'media' &&
+      $variables['element']['#bundle'] === 'document') {
```

Comment:
This should work of course.  You also have the option to choose a more targetted function name and then get rid of the `if` conditional: `localgov_base_croydon_preprocess_field__field_media_document__document()`.